multitenancy/tenant-manager-datamodel/src/main/java/io/apicurio/multitenant/api/beans/ResourceType.java

```diff
@@ -0,0 +1,54 @@
+
+package io.apicurio.multitenant.api.beans;
+
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum ResourceType {
```

Comment:
Are the same strings used in multiple places, i.e. in the enforcement code in the registry? It may be possible that they get out of sync? Maybe add a comment pointing out how the values (strings) are used, in case there is a change or addition of a new one.